service-workers/service-worker/multi-globals/current/test-sw.js

```diff
@@ -0,0 +1,7 @@
+this.addEventListener('fetch', event => {
+    if (event.request.url.includes('test.txt')) {
+        event.respondWith(new Response('current'));
+    } else {
+        event.respondWith(fetch(event.request));
```

Comment:
You could remove these for simplicity: not calling respondWith will fallback to network.